Bpor.Rms/Bpor.Rms.Web/Startup/ConfigurationSetup.cs

```diff
@@ -19,4 +21,13 @@ public static void AddApplicationConfiguration(this ConfigurationManager configu
                 () => new AmazonSecretsManagerClient(RegionEndpoint.GetBySystemName(secretsManagerSettings.Region)));
         }
     }
+
+    public static void ConfigureDatabase(this IServiceCollection services, IConfiguration configuration)
```

Comment:
I would make this name more application specific to avoid confusion with framework extension methods. e.g. `AddParticipantDatabaseContext`